Source/ZoomNet/Models/CallHandlingSettings/CallForwardingChildSubsettings.cs

```diff
@@ -0,0 +1,40 @@
+using System.Text.Json.Serialization;
+
+namespace ZoomNet.Models.CallHandlingSettings
+{
+	/// <summary>
+	/// The call forwarding route settings. It's only required for the <see cref="CallForwardingSubSettings"/>.
```

Comment:
```suggestion
	/// The call forwarding route settings. It's only required for the <see cref="CallForwardingSubsettings"/>.
```

CS1574 XML comment has cref attribute 'CallForwardingSubSettings' that could not be resolved

Subsettings instead of SubSettings. Probably just a typo.

Reply:
Oops, fixed! Thank you!